tests/robot-tests/tests/libs/utilities.py

```diff
@@ -283,4 +293,4 @@ def user_clicks_element_if_exists(selector):
 
 def user_chooses_what_to_do_on_step_failure(selector):
```

Comment:
I know this isn't a change made in your PR, but it seems like this keyword isn't being used and can be removed?

Reply:
Cool I'll have a scan around and sanity check, ta.

Reply:
Yup agree I can't see it being used.  Begone...